config/checkMetroBundlerPort.js

```diff
@@ -0,0 +1,11 @@
+const {isPackagerRunning} = require('@react-native-community/cli-tools');
+
+isPackagerRunning().then((result) => {
+    if (result === 'unrecognized') {
```

Comment:
Sorry I don't quite get what's happening here - can we maybe add some comments or link to documentation? What does `isPackagerRunning()` do and why should we assume there is a port conflict when it returns a string of `unrecognized`?

Reply:
Yes, sorry. I don't think there is any web docs for this function. I got it only from source code. https://github.com/react-native-community/cli/blob/master/packages/tools/src/isPackagerRunning.ts

I can migrate the doc block from source code to our script if that would be enough.

Reply:
>migrate the doc block from source code to our script if that would be enough

Ah nice. Yes, please! I think this would really help make it very clear how this is working.

Reply:
Sorry for the delay. 

I have only just added the comment. Please let me if formatting is ok or any other clarifications are needed.

Thanks.

Reply:
Looks great thanks!